control-center/include/etc/confluent/docker/control-center.properties.template

```diff
@@ -1,54 +1,67 @@
-confluent.controlcenter.command.topic.replication={{env.get('CONTROL_CENTER_COMMAND_TOPIC_REPLICATION', env['CONTROL_CENTER_REPLICATION_FACTOR'])}}
-confluent.metrics.topic.replication={{env.get('CONTROL_CENTER_METRICS_TOPIC_REPLICATION', env['CONTROL_CENTER_REPLICATION_FACTOR'])}}
+{% set metrics_props = {
+    'confluent.metrics.topic': ['CONTROL_CENTER_METRICS_TOPIC', 'CONFLUENT_METRICS_TOPIC'],
+    'confluent.metrics.topic.retention.ms': ['CONTROL_CENTER_METRICS_TOPIC_RETENTION_MS', 'CONFLUENT_METRICS_TOPIC_RETENTION_MS'],
+    'confluent.metrics.topic.partitions': ['CONTROL_CENTER_METRICS_TOPIC_PARTITIONS', 'CONFLUENT_METRICS_TOPIC_PARTITIONS'],
+    'confluent.metrics.topic.skip.backlog.minutes': ['CONTROL_CENTER_METRICS_TOPIC_SKIP_BACKLOG_MINUTES', 'CONFLUENT_METRICS_TOPIC_SKIP_BACKLOG_MINUTES']
+} -%}
 
-{# optional properties #}
-{% set other_props = {
-    'CONTROL_CENTER_ID': 'confluent.controlcenter.id',
-    'CONTROL_CENTER_NAME': 'confluent.controlcenter.name',
-    'CONTROL_CENTER_MONITORING_INTERCEPTOR_TOPIC': 'confluent.monitoring.interceptor.topic',
-    'CONTROL_CENTER_MONITORING_INTERCEPTOR_TOPIC_PARTITIONS': 'confluent.monitoring.interceptor.topic.partitions',
-    'CONTROL_CENTER_MONITORING_INTERCEPTOR_TOPIC_RETENTION_MS': 'confluent.monitoring.interceptor.topic.retention.ms',
-    'CONTROL_CENTER_INTERNAL_TOPICS_PARTITIONS': 'confluent.controlcenter.internal.topics.partitions',
-    'CONTROL_CENTER_INTERNAL_TOPICS_RETENTION_MS': 'confluent.controlcenter.internal.topics.retention.ms',
-    'CONTROL_CENTER_INTERNAL_TOPICS_CHANGELOG_SEGEMENT_BYTES': 'confluent.controlcenter.internal.topics.changelog.segment.bytes',
-    'CONTROL_CENTER_LICENSE': 'confluent.license',
-    'CONTROL_CENTER_CONFLUENT_LICENSE': 'confluent.license',
-    'CONFLUENT_METRICS_TOPIC_PARTITION': 'confluent.metrics.topic.partitions',
-    'CONFLUENT_METRICS_TOPIC_REPLICATION': 'confluent.metrics.topic.replication',
-    'CONFLUENT_SUPPORT_METRICS_ENABLE': 'confluent.support.metrics.enable',
-    'CONFLUENT_METADATA_BOOTSTRAP_SERVER_URLS': 'confluent.metadata.bootstrap.server.urls',
-    'CONFLUENT_METADATA_BASIC_AUTH_USER_INFO':'confluent.metadata.basic.auth.user.info',
-    'PUBLIC_KEY_PATH': 'public.key.path',
-    'CONTROL_CENTER_CPREST_URL': 'cprest.url',
-    'CONFIG_PROVIDERS': 'config.providers',
-    'CONFIG_PROVIDERS_SECUREPASS_CLASS': 'config.providers.securepass.class',
+{% set monitoring_props = {
+    'confluent.monitoring.interceptor.topic': ['CONTROL_CENTER_MONITORING_INTERCEPTOR_TOPIC'],
+    'confluent.monitoring.interceptor.topic.partitions': ['CONTROL_CENTER_MONITORING_INTERCEPTOR_TOPIC_PARTITIONS'],
+    'confluent.monitoring.interceptor.topic.retention.ms': ['CONTROL_CENTER_MONITORING_INTERCEPTOR_TOPIC_RETENTION_MS'],
+    'confluent.monitoring.interceptor.topic.skip.backlog.minutes': ['CONTROL_CENTER_MONITORING_INTERCEPTOR_TOPIC_SKIP_BACKLOG_MINUTES']
```

Comment:
I see a few new ones like this one `confluent.monitoring.interceptor.topic.skip.backlog.minutes`. Any reason for special casing them?

Reply:
It's because they are irregularly translated properties—aka `CONTROL_CENTER_MONITORING_` is translated to `confluent.monitoring.`—that are accepted in c3, but they weren't defined in the template file before. 

So before this PR, if someone specified env_prop `CONTROL_CENTER_MONITORING_INTERCEPTOR_TOPIC_SKIP_BACKLOG_MINUTES`, it will be falsely translated as `confluent.controlcenter.monitoring.interceptor.topic.skip.backlog.minutes`, instead of `confluent.monitoring.interceptor.topic.skip.backlog.minutes`.

This PR adds all the missing irregularly translated properties.

Reply:
Wonderful